definitions/npm/react-router-dom_v5.x.x/flow_v0.63.x-/react-router-dom_v5.x.x.js

```diff
@@ -0,0 +1,171 @@
+declare module "react-router-dom" {
+  declare export var BrowserRouter: React$ComponentType<{|
+    basename?: string,
+    forceRefresh?: boolean,
+    getUserConfirmation?: GetUserConfirmation,
+    keyLength?: number,
+    children?: React$Node
+  |}>
+
+  declare export var HashRouter: React$ComponentType<{|
+    basename?: string,
+    getUserConfirmation?: GetUserConfirmation,
+    hashType?: "slash" | "noslash" | "hashbang",
+    children?: React$Node
+  |}>
+
+  declare export var Link: React$ComponentType<{
+    className?: string,
+    to: string | LocationShape,
+    replace?: boolean,
+    children?: React$Node
+  }>
+
+  declare export var NavLink: React$ComponentType<{
+    to: string | LocationShape,
+    activeClassName?: string,
+    className?: string,
+    activeStyle?: Object,
```

Comment:
Could you replace with `CSSStyleDeclaration`, `Object` is `any` from Flow 0.75.

Reply:
trying CSSStyleDeclaration creates the following error:
```
    Cannot create `NavLink` element because:
     - object literal [1] is incompatible with `CSSStyleDeclaration` [2] in property `activeStyle`.
     - object literal [3] is incompatible with `CSSStyleDeclaration` [4] in property `style`.
    
       test_react-router-dom.js:111:7
                  v-------
       111|       <NavLink
       112|         to="/about"
       113|         activeClassName="active"
       114|         className="link"
       115|         activeStyle={{ color: "red" }}
       116|         style={{ color: "blue" }}
       117|         isActive={(match, location) => true}
       118|         strict
       119|         exact
       120|       >
       121|         About
       122|       </NavLink>;
                  ---------^
    
    References:
       test_react-router-dom.js:115:22
       115|         activeStyle={{ color: "red" }}
                                 ^^^^^^^^^^^^^^^^ [1]
       react-router-dom_v5.x.x.js:28:19
        28|     activeStyle?: CSSStyleDeclaration,
                              ^^^^^^^^^^^^^^^^^^^ [2]
       test_react-router-dom.js:116:16
       116|         style={{ color: "blue" }}
                           ^^^^^^^^^^^^^^^^^ [3]
       react-router-dom_v5.x.x.js:29:13
        29|     style?: CSSStyleDeclaration,
                        ^^^^^^^^^^^^^^^^^^^ [4]
```
`{ +[string]: mixed }` works, which is still an improvement i suppose.

Reply:
Weird, that used to work, but it can't, it's a class... Confused, but yeah `{ +[string]: mixed }` is fine, thanks!